### ISSUE TITLE
New name

### ISSUE THREAD

**Issue by KAMiKAZOW** [polymorphicshade/NewPipe#12]
It's a fork. Don't use the same name. It's not cool.
Call it Ye Olde Pipe or whatever.

**Comment by Madis0** [comment]
BlockPipe, PipeFilter, UnPipe are some that come to mind.

**Comment by m3peror** [comment]
finerpipe

**Comment by test2a** [comment]
Freshpipe ?

**Comment by Eated-Universe** [comment]
NewerPipe

**Comment by LostWanita21** [comment]
I'm in favor of BlockPipe, Or SBPipe, both letting one know what the fork is about right in the title.

**Comment by Lightsockie** [comment]
It's a branch of newpipe with specific improvements merged in. If dev wants to keep the same name I think it's appropriate. It makes sense to do so

**Comment by polymorphicshade** [comment]
@KAMiKAZOW yes I know I need a new name for this, I haven't gotten around to this due to end-of-year work stuff, but I'm free-er now.
I was maybe thinking... SponsorPipe?

**Comment by ajayyy** [comment]
Maybe the logo could be the newpipe triangle in a shield?

**Comment by ajayyy** [comment]
If you want less "sponsor" in the name, "SkipPipe" could work too.

**Comment by Dev-i-l** [comment]
it's ajay the guy who created sponsorblock, and we should wait longer for more creative names

**Comment by connectety** [comment]
> Maybe the logo could be the newpipe triangle in a shield?
<img src="https://user-images.githubusercontent.com/26038710/101332851-2a2fbc80-3876-11eb-8152-4f93628d42f2.png" width="256">

(I have a source Inkscape-SVG file, but I cant upload it in this comment and I dislike having to find temporary solution for it. If you want it ask me for it. Probably you will get a link which will expire)

**Comment by ToughGuyKunio** [comment]
NewPipesOnTheBlock

**Comment by hemanta212** [comment]
NewPipeS could work too

**Comment by Dev-i-l** [comment]
the new logo is dope

**Comment by Atemu** [comment]
Yeah no kidding, it's amazing!

If it had the same tone of red as NP and maybe a similar, subtle shadow, it'd be perfect I think.

>  I was maybe thinking... SponsorPipe?

That name kind of makes it sound like it's NewPipe but with Sponsors, not without them.

BlockPipe and PipeFilter are pretty general but maybe perhaps we'll extend blocking/filtering beyond SponsorBlock in the future and upstream probably won't have any of that.

I'm partial towards PipeFilter personally. It sounds nice and describes this fork's purpose very well: Filtering out the gunk in the pipe.

**Comment by connectety** [comment]
Its SponsorBlocks red and NewPipe's white, so here you have it with NewPipe's more muted red ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683162/muted-red.zip)):
<img src="https://user-images.githubusercontent.com/26038710/101985600-63f32f80-3c89-11eb-9bea-8c53501755fc.png" width="256">

Also...
I thought someone would call me out for it...
Adaptive Icons!!: 

Red Background ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683171/red-bg.zip)):

<img src="https://user-images.githubusercontent.com/26038710/101985733-10351600-3c8a-11eb-8444-b8003bb7f90d.png" width="256">


White Background ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683173/white-bg.zip))):
<img src="https://user-images.githubusercontent.com/26038710/101985779-5c805600-3c8a-11eb-8695-b2aa66347254.png" width="256">


NewPipes Grey Background ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683175/grey-bg.zip)):
<img src="https://user-images.githubusercontent.com/26038710/101985809-83d72300-3c8a-11eb-94da-3da79cbda8bd.png" width="256">

**Comment by Madis0** [comment]
If you rely on a shape, the icon definitely needs the outline because of adaptive icons.

**Comment by ajayyy** [comment]
![Screenshot_20201212-163710.jpg](https://user-images.githubusercontent.com/12688112/101995521-8476b600-3c98-11eb-8c3a-a76a504c342c.jpg)

Here's a conversation with the NewPipe team.

Let's start from scratch with a completely new name and logo

**Comment by Atemu** [comment]
Name I can understand but I think the logo is distinct enough, especially the "white background" one with SB's white line along the border.

The app is 99% NewPipe code afterall, it wouldn't make sense to make the logo suggest it's something entirely different.

**Comment by ajayyy** [comment]
@Atemu If both appear in a search listing on F-droid, it could confuse a user into not knowing which one is the official one

**Comment by Atemu** [comment]
With a different name and a description that clearly states we're a fork (F-Droid search listing would look like "**PipeFilter** A NewPipe fork with SponsorBlock integration."), I can't see how you'd confuse it with "**NewPipe** A free lightweight YouTube frontend for Android".

**Comment by denilsonsa** [comment]
If we go for *SkipPipe* (because that's exactly what it does), we could modify the logo to look like an **S**, like this quick and dirty mockup: 
![image](https://user-images.githubusercontent.com/121676/102009303-d79f4600-3d36-11eb-9c1b-86615c08d301.png)

Otherwise, Pipe is a synonym for Tube, so we could try looking for something in those lines. We could go for *UncloggedTubes*, *CloggedTubes*, *SeriesOfTubes*, or something like that ([wikipedia](https://en.wikipedia.org/wiki/Series_of_tubes), [xkcd](https://xkcd.com/494/)). Or just *Tubular*, meaning [something](https://www.urbandictionary.com/define.php?term=Tubular) [cool](https://en.wiktionary.org/wiki/tubular#Adjective).

**Comment by polymorphicshade** [comment]
@KAMiKAZOW no need to close this issue yet, we still need a new name and logo, per the original NewPipe devs' request.

**Comment by ZaneHannanAU** [comment]
WarpPipe?

**Comment by polymorphicshade** [comment]
So I really like the name _Tubular_. It's unique enough, a bit goofy, and not totally specific to the SponsorBlock additions, leaving room for more additions and enhancements for the app.

Maybe we can tweak the icon a bit more, though my artistic abilities are quite limited...

**Comment by Fatcakes** [comment]
Tubular is good. What are people's thoughts on _Barrel_ and the logo could be the newpipe red with white rifling?

Personally, I like Tubular better.

**Comment by grravity** [comment]
Whatever ends up happening, I just wanted to suggest adding a link to this repo in the about section on the app.

**Comment by nadiration** [comment]
Don't change the name and the logo to something far away from NewPipe's.

**Comment by nadiration** [comment]
> BlockPipe, PipeFilter, UnPipe are some that come to mind. Also the package name should differ.

The package name should definitely differ

**Comment by nadiration** [comment]
> It was SponsorBlocks red and NewPipe's white,
> so here you have it with NewPipe's more muted red ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683162/muted-red.zip)):
> <img alt="" width="256" src="https://user-images.githubusercontent.com/26038710/101985600-63f32f80-3c89-11eb-9bea-8c53501755fc.png">
> 
> Also...
> I thought someone would call me out for it...
> Adaptive Icons!!:
> 
> Red Background ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683171/red-bg.zip)):
> <img alt="" width="256" src="https://user-images.githubusercontent.com/26038710/101985733-10351600-3c8a-11eb-8444-b8003bb7f90d.png">
> 
> White Background ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683173/white-bg.zip))):
> <img alt="" width="256" src="https://user-images.githubusercontent.com/26038710/101985779-5c805600-3c8a-11eb-8695-b2aa66347254.png">
> 
> NewPipes Grey Background ([zipped inkscape svg](https://github.com/polymorphicshade/NewPipe/files/5683175/grey-bg.zip)):
> <img alt="" width="256" src="https://user-images.githubusercontent.com/26038710/101985809-83d72300-3c8a-11eb-94da-3da79cbda8bd.png">

The logo looks more of an adblocker then a media player. NewPipe is more than that. I wouldn't recommend this logo IMO

**Comment by nadiration** [comment]
> > Maybe the logo could be the newpipe triangle in a shield?
> 
> <img alt="" width="256" src="https://user-images.githubusercontent.com/26038710/101332851-2a2fbc80-3876-11eb-8152-4f93628d42f2.png">
> 
> [Source Inkscape-SVG file](https://github.com/polymorphicshade/NewPipe/files/5651738/sponsorPipe.zip) (its zipped because GitHub does not allow to comment SVGs).
> 
> Please only see this as temporary, so we can improvement by criticism.


This logo looks more of an adblocker then a media player. NewPipe is more than that. I wouldn't recommend this logo IMO

**Comment by nadiration** [comment]
> @KAMiKAZOW yes I know I need a new name for this, I haven't gotten around to this due to end-of-year work stuff, but I'm free-er now.
> I was maybe thinking... SponsorPipe?

SponsorPipe looks more of what we we where escaping in the first lol i mean sponsors, ads... When i hear the word sponsor i relate it with ads. Name the fork with something not far from NewPipe like NewPipe S also SponsorBlock is not all the app is so don't stress too much on including that long SponsorBlock in the app name.

**Comment by nadiration** [comment]
> NewPipeS could work too
> 
> ![Screenshot_2020-12-09_10-38-49](https://user-images.githubusercontent.com/25475894/101587035-592b6700-3a0b-11eb-925a-9fe90d21edec.png)

I agree with your name, although there should be a space between the NewPipe and the S, like NewPipe S.
I didn't agree with the logo though it looks more of an adblocker and NewPipe is more than that.

**Comment by Crissomar** [comment]
wipe pipe?
maybe something w/ "free"?
libera tube?
no se ingles y puede que la respuesta se encuentre en otros idiomas 👀

**Comment by grravity** [comment]
I do like the idea of LibrePipe, or LiberaTube.

**Comment by Harsh-dbey** [comment]
It's been a month since this fork's inception, yet it has not been published on f Droid because they need to pick a new name first, I am not saying to rush things, but we shouldn't be too slow either in this process. Also NewPipeS will make a great name if S is displayed distinctly

**Comment by Timmy183-ui** [comment]
It's been a month since this fork's inception, yet it has not been published on f Droid because they need to pick a new name first, I am not saying to rush things, but we shouldn't be too slow either in this process. Also NewPipeS will make a great name if S is displayed distinctly

**Comment by AntiSol** [comment]
How about "BetterPipe"?

**Comment by fikrillah** [comment]
Do we already have new name and logo? Probably vote would quicken the process? Or maybe someone has a better idea? I'm sorry, but i can't wait to see this app on f-droid

**Comment by polymorphicshade** [comment]
@fikrillah for now I'm going with Tubular. The icon will come later. I'm in no rush to publish this on FDroid at the moment (I haven't done that before...) though I do plan to once I finish the current stuff I'm working on. If you need it on FDroid for automatic updates, the current version of this work automatically checks this gitbub repo for updates.

**Comment by nadiration** [comment]
Tubular is not an interesting name IMO

**Comment by Timmy183-ui** [comment]
> @fikrillah for now I'm going with Tubular. The icon will come later. I'm in no rush to publish this on F-Droid at the moment (I haven't done that before...) though I do plan to once I finish the current stuff I'm working on. If you need it on F-Droid for automatic updates, the current version of this automatically checks this gitbub repo for updates.

A surprise to be sure but a welcome one😜
F Droid isn't just the place for updates, it acts like sort of assurance that the app is safe and contains no proprietary junk, that is the reason people ask for f Droid availability

**Comment by samyak039** [comment]
a little late to the party but **NeoPipe** can be a good name, "neo" literally means new and pronounce alike also.

**Comment by alienthief** [comment]
@samyak039 
NeoPipe is pretty nice

**Comment by Tisckkytt** [comment]
NeoPipe is actually good & you can add a S at the end [NeoPipeS] to indicate that it is based on NewPipe & got sponsorblock.

But whichever name & logo is settled, please make sure to make the icon adaptive.

**Comment by avipars** [comment]
NextPipe

**Comment by nadiration** [comment]
@Tisckkytt   why S? NeoPipe is a new name and Sponsorblock isn't all the app is

**Comment by TheEvilSkeleton** [comment]
My personal thoughts:

My favorite name right now: **SponsorPipe**
My favorite logo right now: https://github.com/polymorphicshade/NewPipe/issues/12#issuecomment-739792322

**Comment by Mhowser** [comment]
NeoPipe is the way to go, it doesn't stray far from the original name.

**Comment by hefty-fading-culprit-epiphany-drew** [comment]
@polymorphicshade what's stopping you now? The name is decided. Implement it

**Comment by Mhowser** [comment]
@hefty-fading-culprit-epiphany-drew Patience is a virtue.

**Comment by RezurrectBodg** [comment]
@ghost Well, no, it's not, we have **Tubular** and we have **NeoPipe**

**Comment by samyak039** [comment]
I would suggest you should go with NeoPipe, and these are my reasons:
- NEO means: new, modified, recent. ([dictionary](https://www.dictionary.com/browse/neo))
- there are many software who uses "neo" when they have made a fork or revieved.
  - [neovim](https://neovim.io/): it's an awesome text editor, fork of [vim](https://www.vim.org/)
  - [neomutt](https://neomutt.org/): a terminal based email client, fork of [mutt](http://www.mutt.org/)
- it sounds similar to NewPipe, so not much is changing.

disclaimer: [I suggested "NeoPipe"](https://github.com/polymorphicshade/NewPipe/issues/12#issuecomment-770210827), got the idea from neovim and neomutt 😁

**Comment by Atemu** [comment]
I feel like NeoPipe is too close to NewPipe. It's not like neovim where it's clear that 'neo' was prepended to the old name; the difference to 'NewPipe' would only be one letter here and the meaning is synonymous.

Tubular sounds good too but I still really like @Madis0's recommendation of 'PipeFilter'.

**Comment by ajayyy** [comment]
To clarify on what @Atemu said, the NewPipe team asked for the new name to be completely different

**Comment by TheEvilSkeleton** [comment]
> I feel like NeoPipe is too close to NewPipe

I agree. I literally read "NewPipe" twice as I was reading it.

**Comment by Mhowser** [comment]
> I feel like NeoPipe is too close to NewPipe. It's not like neovim where it's clear that 'neo' was prepended to the old name; the difference to 'NewPipe' would only be one letter here and the meaning is synonymous.
> 
> Tubular sounds good too but I still really like @Madis0's recommendation of 'PipeFilter'.

PipeFilter sounds pretty silly as a name for this app, like imagine recommending it to a person; "hey you should get pipefilter!" "wtf is a pipefilter??"

I'm really rooting for `Neopipe` because it's familiar and sounds nice off the tongue. Plus it will show up nearby Newpipe in app lists, hopefully exposing more users to this superior Fork.

**Comment by RezurrectBodg** [comment]
@Mhowser Maybe chamge the latter part too? Like Neo**pan**, Neo**pile** or smth

**Comment by Mhowser** [comment]
NeoTube

**Comment by Ra-Na** [comment]
Just some proposals: Newtube, NPX (NewPipe eXtended), PimpedPipe

**Comment by nadiration** [comment]
**theTube**

**Comment by JMVS** [comment]
AltPipe

**Comment by AntiSol** [comment]
MeTube

As in "I'm taking back control of my youtube experience by having sponsorblock integration in my player"

**Comment by Ra-Na** [comment]
Fadtube

**Comment by RezurrectBodg** [comment]
@AntiSol associates with MiTube, Mi as in Xiaomi. Otherwise I like it.

**Comment by Mhowser** [comment]
Mytube

**Comment by nadiration** [comment]
**NeoHub**

**Comment by nadiration** [comment]
**Neohub**

**Comment by CounterContradiction** [comment]
I'm thinking, maybe instead of filter, [sieve or sift](url) is a better word?  

So TubeSieve/TubeSift or PipeSieve/PipeSift

**Comment by nadiration** [comment]
**FreeWatch**
It's better to name it something then being stuck

**Comment by pokegamer5547** [comment]
Newatch

**Comment by StarkoPY** [comment]
Plumber

**Comment by Tifter** [comment]
Open Floodgate? 
Open as in open-source. Floodgate for it's more powerful, and it's open :)

**Comment by Tifter** [comment]
Maybe "open pipeline" or just "pipeline"?

**Comment by Tifter** [comment]
But I so like Tubular. Too bad I'm not a logo designer

**Comment by RezurrectBodg** [comment]
I so much **LOVE** Floodgate! "Open" part is not necessary.

**Comment by Mhowser** [comment]
I think we should go with Tubular, Its nice and simple.

**Comment by RezurrectBodg** [comment]
Two really cool and original variants we have on hand - Tubular and Floodgate

**Comment by rm210** [comment]
Welp, Pipe was substitute for Tube and IMO "Tube" is overused a lot(SkyTube etc.). So, to replace Tube with something that holds water, I would suggest Tank or Dam. Hence, NeoTank or NeoDam :P

But +1 for FloodGate as it is used by Dams xD
Also, few icon ideas 

![201](https://user-images.githubusercontent.com/53932738/124743314-3fb7b280-df3b-11eb-8603-58e78296c728.png)
![202](https://user-images.githubusercontent.com/53932738/124743345-47775700-df3b-11eb-9d27-666ea5714851.png)
![203](https://user-images.githubusercontent.com/53932738/124743364-4ba37480-df3b-11eb-8f59-76b2e9d59908.png)
![204](https://user-images.githubusercontent.com/53932738/124744323-44309b00-df3c-11eb-8160-da5c964f4fe0.png)

Basically Play button got flooded.
Ignore the last one(I was trying to present a gate being flooded :D) 
Inspired by [this one](https://www.netclipart.com/pp/m/19-195887_flood-clipart-flooded-house.png)

**Comment by seniorm0ment** [comment]
I like the diest dsign a lot.
Also I'd suggest XMPP over Matrix from a privacy and security stance.

**Comment by login22rvp** [comment]
NewPipeX

**Comment by qwrwed** [comment]
> 
> 
> If we go for _SkipPipe_ (because that's exactly what it does), we could modify the logo to look like an **S**, like this quick and dirty mockup:
> ![image](https://user-images.githubusercontent.com/121676/102009303-d79f4600-3d36-11eb-9c1b-86615c08d301.png)
> 
Fixed some slight asymmetry, uploading it here if anyone else wants to use it to change the icon locally:
![NewPipeS_6](https://user-images.githubusercontent.com/2429893/132316621-d0c1fe00-e654-4f9d-9c3d-8a9840316d27.png)

**Comment by Tifter** [comment]
Sure, Skippipe works, but I would say go for something original. There's some good examples in the comments, I still like Tubular. 

Don't take it personal, but the icon is just lazy and kind of insulting to the newpipe team. It's better with an original name and a generic icon. 

Tubular as the name and a white play button on a black background for example. Change the icon to something more interesting later if necessary. Maybe @rm210 could make something simple

**Comment by Tifter** [comment]
> > If we go for _SkipPipe_ (because that's exactly what it does), we could modify the logo to look like an **S**, like this quick and dirty mockup:
> > ![image](https://user-images.githubusercontent.com/121676/102009303-d79f4600-3d36-11eb-9c1b-86615c08d301.png)
> 
> Fixed some slight asymmetry, uploading it here if anyone else wants to use it to change the icon locally:
> ![NewPipeS_6](https://user-images.githubusercontent.com/2429893/132316621-d0c1fe00-e654-4f9d-9c3d-8a9840316d27.png)

Sure, Skippipe works, but I would say go for something original. There's some good examples in the comments, I still like Tubular.

Don't take it personal, but the icon is just lazy and kind of insulting to the newpipe team. It's better with an original name and a generic icon.

Tubular as the name and a white play button on a black background for example. Change the icon to something more interesting later if necessary. Maybe @rm210 could make something simple?

**Comment by qwrwed** [comment]
I agree that the icon (and name) could definitely be better/more distinct - I've just edited the image I'm using in my launcher, and posted it here because it might save someone else that effort.

**Comment by JacobTravers** [comment]
Here to chip in some ideas: "NeoDuct" as in a wiring duct or aqueduct etc., or maybe "Condu" like a conduit, but also because it sounds like fondue.

**Comment by snezhniylis** [comment]
I don't think any radical change should be performed on the icon. After all, it's the same NewPipe, only with SponsorBlock slapped on top of it.

My proposal is to simply make it look like a fast-forward button ⏩ (for obvious reasons). A quick mock-up:
![NewPipeSB](https://user-images.githubusercontent.com/85341530/135525418-cc3dc890-23d7-4d72-bb9c-a21ded543e76.png)

<details>
<summary>For comparison:</summary>


![NewPipe](https://user-images.githubusercontent.com/85341530/135525834-5a0434ff-9287-42a8-b890-898b3686e62b.png)


</details>

**Comment by snezhniylis** [comment]
Also, I don't think the name should be changed. `NewPipe (SponsorBlock)` or even `NewPipe SB` would do. After all, we're not talking about a full-blown fork. It's the same app with one extra feature.

**Comment by Tifter** [comment]
> I don't think any radical change should be performed on the icon. After all, it's the same NewPipe, only with SponsorBlock slapped on top of it.
> 
> My proposal is to simply make it look like a fast-forward button ⏩ (for obvious reasons). A quick mock-up: ![NewPipeSB](https://user-images.githubusercontent.com/85341530/135525418-cc3dc890-23d7-4d72-bb9c-a21ded543e76.png)
> For comparison:

The fast forward icon idea is really good. But newpipe didn't want us to use their icon and name, so I think we should respect that.

**Comment by snezhniylis** [comment]
> But newpipe didn't want us to use their icon

Their concerns were about the first proposed logo, which looked kind of identical to the original one. The logo I proposed is distinct enough for the end user.

> and name

I'm not sure they were aware about the scope of this fork. It's quite common to have feature forks with very slight name changes. Also, convenient for end users.

> Or maybe a next chapter button icon ⏭️

I thought about this as well. But in my imagination it looked too asymmetrical and flat. It would also lose its magic of resembling the original logo. Though, maybe someone can pull it off.

**Comment by denilsonsa** [comment]
If we go for Tubular, as many people liked, I sketched up a variation of the logo, something that reminds of NewPipe, but seems different enough (in my opinion). I'm 100% sure someone else can draw a better logo than me, or at least draw it cleanly.

![Quick-and-dirty Tubular logo](https://user-images.githubusercontent.com/121676/135574193-500403c5-82b0-4e15-a15c-4ab0c06d6a3f.png)

**Comment by JacobTravers** [comment]
Just gonna bring these 2 notable comments to the end of the thread again as a friendly reminder: the upstream project and the maintainer of this project are in agreement to try and move away from Newpipe's branding.

> I asked the NewPipe team for their opinion:
> ![image](https://user-images.githubusercontent.com/12688112/101996204-47152700-3c9e-11eb-881a-de7984375b23.png) 
> Let's start from scratch with a completely new name and logo

the owner of this repo's [comment](https://github.com/polymorphicshade/NewPipe/issues/12#issuecomment-752182091):

> So I really like the name _Tubular_. It's unique enough, a bit goofy, and not totally specific to the SponsorBlock additions, leaving room for more additions and enhancements for the app.

**Comment by JacobTravers** [comment]
I also made this to chip in, but it's probably still too close to the Newpipe logo... 😅 
![Tubular_App_Icon](https://user-images.githubusercontent.com/46337843/135668260-5d2c8c26-a6cd-4e61-b11d-1aa131813718.png)
I have it in SVG format if it's acceptable or if anybody wants to improve upon it

**Comment by Mhowser** [comment]
@JacobTravers can you make the icon with a blue/green swirling gradient so it looks like a wave? I that will suite the name Tubular very well

**Comment by NSTAdventure** [comment]
> I made my version, what do you think? ![test](https://user-images.githubusercontent.com/24195572/136754224-bfae6c52-225d-4906-b1f0-653786b61d82.png) I would like to put something in the negative space of the first triangle, but I cant think of anything good.

Nyu ~ i love your version so much mate! 🙏😍💖

That was good idea
Also for name my mind say GreenPipe instead of newpipeX or Newpipe+ 
(GreenPipe inspired from your icon version 🥰🥰🥰🥰)

What do you think? 🙏🙂

**Comment by bkqtnte10** [comment]
Name suggestions:
- CleanPipe
- ClearPipe
- [Chenille](https://en.wikipedia.org/wiki/Pipe_cleaner)

**Comment by yephny** [comment]
It's been more than a year. Any updates on this and a possible f-droid release?

**Comment by seniorm0ment** [comment]
It's on Izzy's F-Droid repo btw

**Comment by yephny** [comment]
It is just as many other open-source apps that either do not qualify for or want an F-Droid release sincd Izzy pulls the installation files from Github. I'm asking about this app in the main F-Droid repository where apps are built by F-Droid.

**Comment by jonlandrum** [comment]
Pipe Cleaner (since there's already a reference to pipes, and pipe cleaners scrub all the crud out of your pipe)

**Comment by pc00per** [comment]
PolyPipe?

**Comment by vctls** [comment]
I really like @snezhniylis 's design. It's pretty self explanatory. Maybe colors could be changed just to differentiate it a bit more.
I personally would prefer if this fork kept some branding similarity, I feel it would better reflect what it actually does, and make it easier to find and to explain to other people, rather than completely changing the name and logo.

**Comment by Mesum-Hussain** [comment]
Maybe we should ditch the urge to show the affiliation with NewPipe or with Sponsorblock. 
I think we should give it a completely unique name and logo.

One idea that I am getting is to do something like LBRY did. Just 4 letters,
But it's spelled as Library. 

Maybe we can call our app STRM for Stream? 

I also have some logos in my mind too. I will create them as soon as I get access to a PC. Currently typing this on my phone.

**Comment by joelchrono12** [comment]
I just wanted to say, I absolutely love FloodGate as a new name. I come from the recent post on Reddit and I want to see how grows beyond just a NP fork

**Comment by AdyaGD** [comment]
NewPipe Vanced.

**Comment by polymorphicshade** [comment]
Closed in favor of using [this](https://github.com/polymorphicshade/NewPipe/discussions/164) discussion.

**Comment by nsde** [comment]
NeoPipe

**Comment by watcherobt** [comment]
YouSkip
